src/screens/HyperSwitch/Analytics/DownloadReportModal.res

```diff
@@ -80,8 +80,15 @@ let make = (~reportModal, ~setReportModal, ~entityName) => {
     },
   }->Identity.genericTypeToJson
 
+  let category = switch entityName {
+  | PAYMENT_REPORT => "Payment"
+  | REFUND_REPORT => "Refund"
+  | DISPUTE_REPORT => "Dispute"
+  | _ => ""
```

Comment:
do we need this default value? if there is no warning you can remove it.

Reply:
it do shows a warning